apm/docs/metrics.md

```diff
@@ -647,6 +625,93 @@ The value of the Metric variable up is zero
 </details>	
 
 
+#### Jaeger Agent Status
+- type: Jaeger
+- description: Jaeger agent status
+<details>
+<summary>Legende details</summary>
+
+##### active
+Number of Jaeger-agent services in up status
+```
+count(up{job="jaeger_agent"} == 1) 
```

Comment:
too many panels can be a distraction, monitoring is about identifying risks, and we don't need to spend too much effort focusing on healthy nodes.

Reply:
This is still important for checking the status of the components.